androidApp/src/main/java/com/mbta/tid/mbta_app/android/onboarding/OnboardingScreenView.kt

```diff
@@ -149,25 +153,31 @@ fun OnboardingScreenView(
                 }
             }
         }
-        OnboardingScreen.HideMaps -> {
+        OnboardingScreen.MapDisplay -> {
+            var localHideMapsSetting by rememberSaveable { mutableStateOf(true) }
```

Comment:
I don't _love_ this being stored locally here, but it is more awkward on iOS. storing a local value & saving on continue seemed like the best approach to me since we want the toggle defaulted to the off position, which is the opposite of the actual default setting value. Definitely welcome any suggestion for a cleaner approach!

Reply:
The only other option I can think of would be setting the setting to true when the screen first appears, but if that’s not instant and there’s a perceptible moment where the switch is on before it flips to off, that’s not worth doing.

Reply:
I tried that first w/ a LaunchedEffect and there was a flicker, will stick with this then!